cmd/openshift-install/internal_integration_test.go

```diff
@@ -271,6 +335,8 @@ func ignitionStorageCmp(ts *testscript.TestScript, neg bool, args []string) {
 	eData, err := os.ReadFile(eFilePathAbs)
 	ts.Check(err)
 
+	eData = replaceReleaseImage(string(eData))
```

Comment:
Isn't this handled by the call to `replaceReleaseImage()` in `expandFile()`?

Reply:
All the other testcases, when `unconfiguredIgnCmp` is called which internally invokes `ignitionStorageCmp` thus calls `replaceReleaseImage()`, those testcases also calls `expandFile()` but a testcase such as  https://github.com/openshift/installer/blob/master/cmd/openshift-install/testdata/agent/unconfigured-ignition/validations/missing-cluster-image-set-allowed.txt doesn't call `expandFile()` so we need to make replacement in the `ignitionStorageCmp` as well.